dwbb/src/interface/DW02_mult.scala

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW02_mult
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(AWidth: Int = 8, BWidth: Int = 16)
```

Comment:
```suggestion
case class Parameter(aWidth: Int = 8, bWidth: Int = 16)
```

Reply:
Changed. Also other unmerged PRs. For merged PRs, can change in the future if necessary.